src/components/bulkConcepts/component/ActionButtons.jsx

```diff
@@ -63,7 +63,12 @@ export class ActionButtons extends Component {
           type="submit"
           className="btn btn-sm mb-1 actionaButtons"
           id="add-button"
-          onClick={() => this.addConceptButton()}
+          onClick={() => this.addConceptButton(
+            this.props.id,
+            this.props.preview.url,
```

Comment:
Is this also about testing the untested areas?

Reply:
Yes, this is part of the first bullet point that handles restructuring of the code in order for it to be tested. The function was formally untested and after the change, I was able to test it.